core/src/test/java/org/apache/iceberg/TestRowDelta.java

```diff
@@ -896,17 +896,14 @@ public void testAddDeleteFilesMultipleSpecs() {
     Map<String, String> summary = snapshot.summary();
 
     Assert.assertEquals(
-        "Should change 4 partitions", "4", summary.get(CHANGED_PARTITION_COUNT_PROP));
+        "Should change 3 partitions", "3", summary.get(CHANGED_PARTITION_COUNT_PROP));
```

Comment:
the test does makes the table temporarily unpartitioned and this was wrongly counted here

Reply:
Why was it counted?